### PR TITLE
NO-JIRA: [doc] fix auto_accept typo

### PR DESCRIPTION
**Pull request by middagj**
I assume this is just a type, since the line above stated `auto_accept` and the parameter as well.

### CODE REVIEW

File: python/proton/_handlers.py [hunk 1]
```diff
@@ -235,8 +235,8 @@ class IncomingMessageHandler(Handler, Acking):
     events related to incoming i.e. received messages.
 
     :type auto_accept: ``bool``
-    :param auto_settle: If ``True``, settle all messages (default). Otherwise
-        messages must be explicitly settled.
+    :param auto_accept: If ``True``, accept all messages (default). Otherwise
+        messages must be explicitly accepted.
```
Comment (astitcher) [inline python/proton/_handlers.py:239]: Fix on your fix! I think the language from MessagingHandler is better "messages must be individually accepted or rejected"

**Review by astitcher — APPROVED** [review]

File: python/proton/_handlers.py (hunk 1 above)
Comment (middagj) [reply python/proton/_handlers.py]: Yes, thanks!

**Comment by astitcher** [comment]
@middagj If you squash these changes I'll be happy to merge the PR.

**Comment by middagj** [comment]
@astitcher Done!

**Pull request merged**